backend/README.md

```diff
@@ -2,7 +2,10 @@
 
 ## Prerequisites
 
--   python 3.x
+-   python 3.10.x
+    -   Previous versions may work, but are not yet checked
```

Comment:
we can later set up poetry to manage our python dependancies. only reason I can currently think of to use older python version for older python libs.

Reply:
Yes, we should talk about that sometime. 
I added the 3.10 because older python versions may not have all python features we currently use, which came up during testing. 
E.g. typing Literals (you commented on one in the config.py) were introduced in 3.8. If you tried to run the program with an older version, an ImportError would be thrown. In this example, they can be used in previous versions with an external package, but then they have to be improved separately / the code has to be changed. 

[typying.Literal python doc](https://docs.python.org/3/library/typing.html#typing.Literal) for reference.